ipld/read.go

```diff
@@ -35,7 +36,7 @@ func RetrieveData(
 	}
 	errGroup, ctx := errgroup.WithContext(parentCtx)
 	errGroup.Go(func() error {
-		return fillQuadrant(ctx, quadrant, dag, dataSquare)
+		return fillQuadrant(ctx, quadrant, merkledag.NewSession(ctx, dag), dataSquare)
```

Comment:
Huh, didn't you already introduce this a few months ago?

Reply:
In core? Yes. This is not the core.